src/webgpu/api/validation/capability_checks/limits/maxColorAttachmentBytesPerSample.spec.ts

```diff
@@ -184,26 +195,19 @@ g.test('createRenderPipeline,at_over')
           code
         );
       },
-      { maxColorAttachments: maxColorAttachmentsLimitMode }
+      kExtraLimits
     );
   });
 
 g.test('createRenderPipelineAsync,at_over')
```

Comment:
This test is almost identical to the one above aside from the final expectation check. We might be able to coalesce them into one and have an addition param `.combine('async', [true, false])`.

Reply:
done